inc/tag.class.php

```diff
@@ -0,0 +1,88 @@
+<?php
+/*
+ * @version $Id: HEADER 15930 2011-10-30 15:47:55Z tsmr $
+ -------------------------------------------------------------------------
+ Mreporting plugin for GLPI
+ Copyright (C) 2003-2011 by the mreporting Development Team.
+
+ https://forge.indepnet.net/projects/mreporting
```

Comment:
All plugin files were affected.
Maybe it's better to do that with another "fix" ?